pom.xml

```diff
@@ -1,147 +1,163 @@
-                    <dependency>
-                        <groupId>org.codehaus.gmaven.runtime</groupId>
-                        <artifactId>gmaven-runtime-2.0</artifactId>
-                        <version>1.5</version>
-                    </dependency>
-                </dependencies>
-                <executions>
-                    <execution>
-                        <goals>
-                            <goal>compile</goal>
-                            <goal>testCompile</goal>
-                        </goals>
-                    </execution>
-                </executions>
-            </plugin>
-            <plugin>
-                <groupId>org.apache.maven.plugins</groupId>
-                <artifactId>maven-surefire-plugin</artifactId>
-                <version>2.18.1</version>
-                <configuration>
-                    <includes>
-                        <include>**/*Spec.java</include>
-                        <include>**/*Test.java</include>
-                    </includes>
-                    <parallel>classes</parallel>
-                    <threadCount>4</threadCount>
-                </configuration>
-            </plugin>
-        </plugins>
-    </build>
+	</dependencies>
+
+	<properties>
+		<project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
+		<start-class>pl.confitura.jelatyna.Application</start-class>
+		<java.version>1.8</java.version>
+		<maven.compiler.source>1.8</maven.compiler.source>
+		<maven.compiler.target>1.8</maven.compiler.target>
+		<lambda-behave.version>0.3</lambda-behave.version>
+		<assertj.version>1.7.0</assertj.version>
+		<h2.version>1.4.182</h2.version>
+		<lombok.version>1.14.8</lombok.version>
+		<spock.version>1.0-groovy-2.0</spock.version>
+		<groovy.version>2.0.1</groovy.version>
+	</properties>
+
+	<build>
+		<plugins>
+			<plugin>
+				<groupId>org.springframework.boot</groupId>
+				<artifactId>spring-boot-maven-plugin</artifactId>
+			</plugin>
+			<plugin>
+				<groupId>org.codehaus.gmaven</groupId>
+				<artifactId>gmaven-plugin</artifactId>
+				<version>1.5</version>
+				<configuration>
+					<providerSelection>2.0</providerSelection>
+				</configuration>
+				<dependencies>
+					<dependency>
+						<groupId>org.codehaus.gmaven.runtime</groupId>
+						<artifactId>gmaven-runtime-2.0</artifactId>
+						<version>1.5</version>
+					</dependency>
+				</dependencies>
+				<executions>
+					<execution>
+						<goals>
+							<goal>compile</goal>
+							<goal>testCompile</goal>
+						</goals>
+					</execution>
+				</executions>
+			</plugin>
+			<plugin>
+				<groupId>org.apache.maven.plugins</groupId>
+				<artifactId>maven-surefire-plugin</artifactId>
+				<version>2.18.1</version>
+				<configuration>
+					<includes>
+						<include>**/*Spec.java</include>
+						<include>**/*Test.java</include>
+					</includes>
+					<parallel>classes</parallel>
+					<threadCount>4</threadCount>
+				</configuration>
+			</plugin>
+		</plugins>
+	</build>
```

Comment:
chyba nie ma co pisać po angielsku :)
Dobry pytanie, gadaliśmy o tym  z Kubą i w sumie to nie wiem na czym stanęło.
W projektach zawsze używałem spacji, ale przekonał mnie w miare argument Kuby, że jak jest tabulacja to każdy sobie ustawi taki rozmiar jaki mu wygodnie i po kłopocie.

A Ty jak uważasz?